src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -156,11 +156,15 @@ public function current_node() {
 	 *
 	 * @see https://html.spec.whatwg.org/#has-an-element-in-the-specific-scope
 	 *
-	 * @param string   $tag_name         Name of tag check.
-	 * @param string[] $termination_list List of elements that terminate the search.
+	 * @param string   $tag_name             Name of tag check.
+	 * @param string[] $termination_list     List of elements that terminate the search.
+	 * @param string   $termination_behavior Optional. "include" or "exclude". If "include"
```

Comment:
I'm a bit uneasy adding new semantics to one of the algorithms defined in the spec. I might be hypocritical on this because maybe I proposed it before.

at least from now, it seems like this is blurring the function just for SELECT, and maybe I'm reading the spec wrong, but it seems like the algorithm for SELECT is much different and much easier: if walking up the stack, and you find any element other than `OPTGROUP` or `OPTION`, then you don't have that element in SELECT scope.

did you consider special-casing the SELECT scope and leaving this function alone?

Reply:
Reverted.